shared/src/main/kotlin/pl/edu/pw/ia/shared/security/SecurityConfig.kt

```diff
@@ -46,6 +47,9 @@ class SecurityConfig(
 						.pathMatchers(swaggerPath).permitAll()
 						.pathMatchers("/webjars/swagger-ui/**").permitAll()
 						.pathMatchers("/actuator/**").permitAll()
+						.pathMatchers(HttpMethod.GET, "/api/v1/threads/**").permitAll()
```

Comment:
I don't like that it's hardcoded in here
But it's only temporary so it's ok xD

Normally we should define an interface eg. 
`AuthorizedExchange` and here we would inject `List<AutorizedExchange>` and iterate over it